region/maridia/outer/Boyon Gate Hall.json

```diff
@@ -1398,6 +1398,87 @@
       "farmCycleDrops": [{"enemy": "Zebbo", "count": 1}],
       "flashSuitChecked": true
     },
+    {
+      "link": [4, 4],
+      "name": "Open Gate, R-Mode Spark Interrupt",
+      "requires": [
+        {"obstaclesCleared": ["R-Mode"]},
+        {"obstaclesCleared": ["A"]},
+        {"refill": ["Energy"]},
+        {"or": [
+          {"canShineCharge": {"usedTiles": 15, "openEnd": 2}},
+          {"and": [
+            {"or": [
+              {"and": [
+                "h_trickyFrozenEnemyRunway",
+                {"enemyKill": {"enemies": [["Boyon"]]}}
+              ]},
+              {"and": [
+                "h_frozenEnemyRunway",
+                "Morph"
+              ]}
+            ]},
+            {"canShineCharge": {"usedTiles": 18, "openEnd": 1}}
+          ]}
+        ]},
+        {"autoReserveTrigger": {"maxReserveEnergy": 95}},
+        "canRModeSparkInterrupt"
+      ],
+      "resetsObstacles": ["R-Mode"],
+      "flashSuitChecked": true,
+      "blueSuitChecked": true,
+      "note": [
+        "Fill up on the respawning Zebbos. Shinecharge under the open gate, then jump into the left Boyon to interrupt.",
+        "A frozen Zebbo can add to the runway if needed."
+      ]
+    },
+    {
+      "link": [4, 4],
+      "name": "R-Mode Spark Interrupt",
+      "requires": [
+        {"obstaclesCleared": ["R-Mode"]},
+        {"refill": ["Energy"]},
+        "canShinechargeMovementTricky",
+        {"or": [
+          "canWalljump",
+          {"and": [
+            "SpaceJump",
+            "HiJump"
+          ]},
+          {"and": [
+            "HiJump",
+            "canSpringBallJumpMidAir"
+          ]}
+        ]},
+        {"or": [
+          {"and": [
+            {"doorUnlockedAtNode": 1},
+            {"doorUnlockedAtNode": 2},
+            {"canShineCharge": {"usedTiles": 13, "openEnd": 1}}
+          ]},
+          {"and": [
+            {"or": [
+              {"doorUnlockedAtNode": 1},
```

Comment:
With [1] open, I think you get something closer to 12+1.  You still need to shortcharge right to left, into the open door now.